apps/web/components/booking/BookingListItem.tsx

```diff
@@ -820,34 +841,32 @@ type NoShowProps = {
   isBookingInPast: boolean;
 };
 
-const Attendee = (attendeeProps: AttendeeProps & NoShowProps) => {
-  const { email, name, bookingUid, isBookingInPast, noShow: noShowAttendee, phoneNumber } = attendeeProps;
+type setAttendeeList = {
+  setAttendeeList: Dispatch<SetStateAction<AttendeeList[]>>;
+};
+
+const Attendee = (attendeeProps: AttendeeProps & NoShowProps & setAttendeeList) => {
+  const { email, name, bookingUid, isBookingInPast, noShow, phoneNumber, setAttendeeList } = attendeeProps;
   const { t } = useLocale();
 
-  const [noShow, setNoShow] = useState(noShowAttendee);
   const [openDropdown, setOpenDropdown] = useState(false);
   const { copyToClipboard, isCopied } = useCopy();
 
   const noShowMutation = trpc.viewer.markNoShow.useMutation({
```

Comment:
@sanjeevs9 @Praashh why dont we move **noShowMutation** to BookingListItem component and reuse the same mutation by passing as props

Reply:
Yes, you are right, but it's better to use this in `NoShowAttendeesDialog` for simplicity. Also, the issue was inside the NoShowAttendeesDialog; the Attendee component's noShow was working fine previously.

Reply:
@Praashh so i have to fix the naming convention only everything else is fine?

Reply:
yeah for now only changes I suggested, but I'll give it a final touch